server/services/interventionsService.test.ts

```diff
@@ -991,3 +1051,41 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 })
+
+describe('serializeDeliusServiceUser', () => {
+  const interventionsService = new InterventionsService(config.apis.interventionsService)
+
+  it('transforms a DeliusServiceUser into a format expected by the Interventions Service', () => {
+    const deliusServiceUser = {
+      otherIds: {
+        crn: 'X123456',
+        nomsNumber: '',
+        mostRecentPrisonerNumber: '',
+      },
+      offenderProfile: {
+        offenderLanguages: {
+          primaryLanguage: 'English',
+          requiresInterpreter: false,
+          otherLanguages: ['French'],
+        },
+      },
+      title: 'Mr',
+      firstName: 'Alex',
+      surname: 'River',
+      dateOfBirth: '01/01/1980',
+      gender: 'Male',
+      ethnicity: 'British',
+      religionOrBelief: 'Agnostic',
+      disabilities: 'Autism',
```

Comment:
question on disabilities.- i thought these came from delius as a list of objects with start and end dates in?

Reply:
ah yeah, you're right, will fix that.